android/build.gradle

```diff
@@ -14,9 +14,9 @@ android {
         applicationId "com.trikot.sample"
         versionCode 1
         versionName "1.0"
-        archivesBaseName = "PatronProjectName-$versionCode"
+        archivesBaseName = "TrikotFrameworkName-$versionCode"
```

Comment:
I think this should not be renamed. It is renamed by boilerplate-setup.sh.

Reply:
Thats the value we found not being renamed during our integration of Jasper. The replacement value is `TrikotFrameworkName`.